### DISCUSSION TITLE
Login to application using instagram #370

### DISCUSSION THREAD

**Discussion by szrtGreg** [DevAdventCalendar/DevAdventCalendar#384] [Ideas] [answered]
Good Morning DAC,
I have found interesting issue. Instagram is not recommending login via its API. **Instead, use Facebook Authentication.**

![obraz](https://user-images.githubusercontent.com/27915290/94254010-6e254d80-ff26-11ea-9552-cd791b3f5666.png)

Source: https://www.instagram.com/developer/

![obraz](https://user-images.githubusercontent.com/27915290/94253953-56e66000-ff26-11ea-9d99-b17a2efe2794.png)

Source: https://developers.facebook.com/docs/instagram-basic-display-api

**Comment by WTobor** [comment, marked as answer]
@DevAdventCalendar/dac-mentoring-group - do you have anything to add in that topic? :)

**Reply by WTobor** [reply]
As we discussed earlier - let's consider it out of scope and close issue ;)
